### DISCUSSION TITLE
【 修正希望 】Karabiner-Elements のタグが karabiner-Elements（語頭が小文字）になってしまっている

### DISCUSSION THREAD

**Discussion by MANABU0SEC** [increments/qiita-discussions#573] [要望・アイデア・不具合]
素晴らしいプラットフォームの運営をありがとうございます！ いつもお世話になっております。

タイトル通りなのですが、Karabiner-Elements のタグが karabiner-Elements（語頭が小文字）になってしまっているため、適切なタグをつけることができません。Karabiner-Elements（語頭も大文字）に修正していただくことは可能でしょうか？ ユーザー側からは修正できないようです。そして願わくば、これまでにこのタグで投稿された記事も、修正後のタグにアップデートされると嬉しいです。

よろしくお願い申し上げます。

**Comment by MANABU0SEC** [comment]
申し訳ありません、投稿したあとで「タグの名寄せリクエスト」のアナウンスに気づきました。このディスカッションはクローズして、そちらの方にコメントいたします。

**Reply by tanaka-asami415** [reply]
@MANABU0SEC 
ありがとうございます🙏
対応が完了しましたら、改めて https://github.com/increments/qiita-discussions/discussions/237 でご報告いたします！
